typescript/cli/src/deploy/warp.ts

```diff
@@ -1026,7 +1034,27 @@ export async function getSubmitterByStrategy<T extends ProtocolType>({
             return new EV5FileSubmitter(metadata);
           },
         },
-        ...createAltVMSubmitterFactories(multiProvider, altVmSigners, chain),
+        [protocol]: {
+          jsonRpc: () => {
+            const { type } = strategyToUse.submitter;
+            assert(
+              type === 'jsonRpc',
+              `Invalid metadata type: ${type}, expected ${TxSubmitterType.JSON_RPC}`,
+            );
+
+            assert(strategyToUse.submitter.privateKey, '');
+            return getProtocolProvider(protocol).createSubmitter(
```

Comment:
This `createSubmitter()` method feels weird. Maybe we should reconsider how submitters are represented in `provider-sdk`. For now, we could just init call `new AltVmJsonSubmitter()` here or wrap it in a function exported from `provider-sdk`.